poc/src/main.rs

```diff
@@ -177,9 +209,9 @@ async fn process_input(
                 println!("Beginning DB INSERT benchmark of {} users", num_users);
 
                 let mut values: Vec<String> = vec![];
-                for _ in 0..*num_users {
+                for i in 0..*num_users {
                     values.push(
-                        thread_rng()
+                        StdRng::seed_from_u64(i)
```

Comment:
It's fine to change this, but why are we doing it?

Reply:
Ah so the reason is for repeatability and fairer comparisons when running the benchmarks between diffs. Without a fixed seed, the usernames generated between runs differ and can lead to a different node state count despite the same run parameters. Does that make sense?

Reply:
Yeah that's all good. Thanks